lib/images.sh

```diff
@@ -11,8 +11,8 @@ elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/}
   export IMAGE_USERNAME=${IMAGE_USERNAME:-core}
 elif [[ "${IMAGE_OS}" == "Centos" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-CentOS-7-x86_64-GenericCloud-1907.qcow2}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-http://cloud.centos.org/centos/7/images}
+  export IMAGE_NAME=${IMAGE_NAME:-CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://cloud.centos.org/centos/8/x86_64/images/}
```

Comment:
why isn't value the same as in vm-setup/roles/v1aX_integration_test/vars/main.yml?

Reply:
I replaced the previous value of centos-update to be the same, see: https://github.com/metal3-io/metal3-dev-env/pull/264/files#diff-ef7461d6dcc137f60cbd6cc5f23fbef5R29

Reply:
Ah sorry, my bad